src/components/dialogmote/DialogmoteDatoField.tsx

```diff
@@ -1,35 +1,15 @@
 import React from "react";
 import { Field } from "react-final-form";
-import { toDatePrettyPrint } from "@/utils/datoUtils";
-import { Datepicker, isISODateString } from "nav-datovelger";
+import { Datepicker } from "nav-datovelger";
 import dayjs from "dayjs";
 import { Label, SkjemaelementFeilmelding } from "nav-frontend-skjema";
+import { validerDato } from "@/utils/valideringUtils";
 
 const texts = {
   datoLabel: "Dato",
   datoPlaceholder: "dd.mm.åååå",
 };
 
-export const validerDatoField = (
```

Comment:
Flyttet denne til valideringUtils for å kunne bruke den både herfra og i avventer-frist-datovalideringen